isis/src/control/apps/cnetcheck/cnetcheck.cpp

```diff
@@ -635,7 +635,12 @@ namespace Isis {
           createdCamera = false;
         }
 
-        QList< ControlMeasure * > measures = cnet.GetMeasuresInCube(serialNumber);
+        QList< ControlMeasure * > measures;
+        if (ignore) {
+          measures = cnet.GetValidMeasuresInCube(serialNumber);
+        } else {
+          measures = cnet.GetMeasuresInCube(serialNumber);
+        }
```

Comment:
Should we also ignore the point if all measures are ignored? That is, can all the measures be set to ignored but not the point? If so, in that case also ignore the point automatically? Or does cnetcheck already report points with no measures?